src/passwordless/client.py

```diff
@@ -0,0 +1,223 @@
+from requests import Request, Response, Session
+from typing import List
+from abc import ABC, abstractmethod
+import logging
```

Comment:
Looks like `isort` may not have run on this file. `import logging` should have gone above lines 1-3 and lines 1-3 are out of order.